ImageInLib/ImageInLib/include/common_functions.h

```diff
@@ -45,6 +45,15 @@ extern "C" {
 
 		dataType ** imageDataPtr; // Image Data Containers
 	} Image_Data;
+
+	// Generate Random Points
+	typedef struct {
+		size_t k, xd, p;
+	}RandomPoints;
+
+	typedef struct {
+		size_t xd, p;
+	}Random2dPoints;
```

Comment:
Give a more clear name i.e. Random3DPoints and Random2DPoints or just use one general structure RandomPoints for both cases.